bin/docserv-createconfig

```diff
@@ -27,6 +27,8 @@ import re
 import sys
 import pygit2
 from xml.etree import cElementTree as etree
+from lxml import etree as etree_for_parsing
```

Comment:
Since we already hat an etree imported, I didn't want to confuse mine with the one already there. 

Solution to this was to get rid of the old one and use **just** `lxml`. Thanks  @tomschr